github/view/GitApplication.kt

```diff
@@ -0,0 +1,198 @@
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package github.view
+
+//import androidx.compose.foundation.layout.
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Button
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.TabRowDefaults.Divider
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.Window
+import androidx.compose.ui.window.WindowPlacement
+import androidx.compose.ui.window.application
+import androidx.compose.ui.window.rememberWindowState
+import kotlinx.coroutines.CoroutineScope
+import github.state.Downloader
+import github.state.Migrator
+import github.state.GlobalState
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+object GitApplication {
+    private val coroutineScope = CoroutineScope(Dispatchers.Default)
+
+    private var repoTextFieldValue by mutableStateOf("vaticle/typedb")
+    private var exploreButtonTextValue by mutableStateOf("Explore $repoTextFieldValue")
+    private var outputTextFieldValue by mutableStateOf("")
+
+    private var smallTextFieldModifier = Modifier.height(50.dp).width(200.dp)
+    private var smallButtonModifier = Modifier.height(50.dp).width(100.dp)
+
+    @JvmStatic
+    fun main(args: Array<String>) {
+        application(exitProcessOnExit = false) {
+            MaterialTheme {
+                mainWindow(::exitApplication)
+            }
+        }
+    }
+
+    @Composable
+    private fun mainWindow(exitApplicationFn: () -> Unit) {
+        Window(
+            title = "Git Application",
```

Comment:
This PR itself has led to this name, which I think sounds more appealing
```suggestion
            title = "GitHub Data Explorer",
```

Reply:
Also, once it's changed, ensure the screenshot is updated (the screenshot has the window title in it).